src/main/java/org/springframework/data/elasticsearch/core/query/Query.java

```diff
@@ -21,6 +21,7 @@
 import java.util.List;
 import java.util.Optional;
 
+import org.elasticsearch.search.builder.SearchSourceBuilder;
```

Comment:
no Elasticsearch imports here. This class must be agnostic of the used client implementation.

Reply:
why can there be no elasticsearch imports if it is in the elasticsearch core package?